config/initializers/good_job.rb

```diff
@@ -13,6 +13,12 @@
       class: "GoodJobStatsDJob",
       set: { priority: 10 },
       description: "Sending GoodJob metrics to statsd every 15s"
+    },
+    mfa_usage_stats: {
+      cron: "every hour",
```

Comment:
I don't have much context on good job but I'm assuming it'll know to run it every hour through the value of the string?

Reply:
Yes, it uses a gem called `fugit` which is [pretty clever at parsing things](https://github.com/floraison/fugit#parse_cronish-and-do_parse_cronish).